Cython/Utility/ModuleSetupCode.c

```diff
@@ -308,6 +315,13 @@
   #ifndef CYTHON_AVOID_BORROWED_REFS
     #define CYTHON_AVOID_BORROWED_REFS 0
   #endif
+  // CYTHON_AVOID_THREAD_UNSAFE_BORROWED_REFS - Avoid borrowed references that are not thread-safe in the free-threaded build of CPython.
+  #if CYTHON_COMPILING_IN_CPYTHON_NOGIL
```

Comment:
```suggestion
  #if CYTHON_COMPILING_IN_CPYTHON_FREETHREADING
```

We changed the macro just because `nogil` was looking ambiguous with Cython's `nogil` feature in other places